package.json

```diff
@@ -24,7 +24,7 @@
     "backbone": "~1.1.2",
     "express": "~4.11.0",
     "jquery": ">=1.11.1",
-    "jsdom": "~3.1.0",
+    "jsdom": ">=3.0.0",
```

Comment:
can you update the engines field also? that should communicate which versions of node to support. https://docs.npmjs.com/files/package.json#engines